src/bin/zmcontrol

```diff
@@ -6,7 +6,6 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
 use strict;
-use warnings;
```

Comment:
Why? :eyes:

Reply:
IntelliJ added it not me

Reply:
Got it, I think its okay to get rid of it in production code. 
A quick google search defines this as : Warning in Perl is the most commonly used Pragma in Perl programming and is used to catch 'unsafe code'.